Tools/Scripts/webkitpy/benchmark_runner/data/plans/motionmark1.3-tentative.plan

```diff
@@ -0,0 +1,252 @@
+{
+    "timeout": 1800,
+    "count": 1,
+    "github_source": "https://github.com/webkit/MotionMark/tree/e15bbf1e075007a3e20d1bd56a97b80b812c48b8/MotionMark",
+    "webserver_benchmark_patch": "data/patches/webserver/MotionMark1.3-tentative.patch",
+    "webdriver_benchmark_patch": "data/patches/webdriver/MotionMark1.3-tentative.patch",
+    "signpost_patch": "data/patches/signposts/MotionMark1.3-tentative.patch",
+    "entry_point": "index.html",
+    "subtest_entry_point": "developer.html",
+    "config": {
+        "orientation": "landscape"
+    },
+    "output_file": "motionmark.result",
+    "github_subtree": {
+        "url": "https://api.github.com/repos/webkit/MotionMark/tree/e15bbf1e075007a3e20d1bd56a97b80b812c48b8/",
```

Comment:
Should be `https://api.github.com/repos/WebKit/MotionMark/git/trees/db6987eb65f3be06ad409c50fe9807845e2e2123`.